### PR TITLE
Make strict to array item and add first and last mode for strict string item

### PR DESCRIPTION
**Pull request by fengmk2**
https://github.com/koajs/querystring-strict/issues/1#issuecomment-76502415

### CODE REVIEW

**Comment by fengmk2** [comment]
@dougwilson @dead-horse

File: README.md [hunk 1]
```diff
@@ -54,12 +54,40 @@ If you want to use this mode, don't use this module.
...
-This mode make `this.query.foo` return strict `string`. Disable multi values.
+This mode make `this.query.foo` return strict `array`.
 
 ```js
 require('koa-qs')(app, 'strict')
 ```
 
+#### What's different
+
+A normal request `GET /foo?p=a,b`.
```
Comment (dead-horse) [inline README.md]: `GET /foo?p=a&q=foo&q=bar` is more clearly?

**Comment by dougwilson** [comment]
This seems fine. I'm not sure `last` is really an option that would make sense (typically if sites exhibit the "last" behavior it's on accident because the authors did not think about the behavior and/or are using a library that did the same).

File: README.md (hunk 1 above)
Comment (fengmk2) [inline README.md]: fix

**Comment by fengmk2** [comment]
@dougwilson yep, let's make it simple, remove `last`.

**Comment by fengmk2** [comment]
now available modes: extended, simple, strict, first.

**Comment by dead-horse** [comment]
LGTM

**Comment by fengmk2** [comment]
I publish a new version.

**Pull request merged**